main.go

```diff
@@ -142,6 +149,44 @@ func DedupGameList(games []string) []string {
 	return result
 }
 
+func concatNotes(game string, notes string) string {
+	// Split moveList by pereiods then by spaces
+	prevMoveList := strings.Split(strings.Join(strings.Split(strings.Split(game, "\"]\n\n")[1], ". "), " "), " ")
```

Comment:
What does this do? And is there no more efficient way to do it? Each split and join creates entirely new strings.